### ISSUE TITLE
Import from the username/sessions screen (For VC's)

### ISSUE THREAD

**Issue by mogmismo** [mogmismo/pfs-sessiontracker-userscript#2]
Michael Eshleman asks,

If you get it worked out, I'd love the ability to export sessions from 'https://secure.paizo.com/people/<=username=>/sessions'

I'm not really programming-savvy though, so dumb it down for me if you can.  :)

**Comment by saekort** [comment]
At the moment the tracker has a toggle called `public reporting` if it is set then others can report for you, if it isn't then others cannot. I believe (off the top of my head) that the default is that you can. This to keep everything a bit accessible.

What I could do is try to let people report for others, but block it when that person has public reporting turned off.

Oh and yeah no problem on the one datastream, I would actually prefer that. :)

**Comment by mogmismo** [comment]
Sounds good...  I've got (yet another) big weekend of gaming, so I'll get back to this early next week, most likely.

**Comment by mogmismo** [comment]
Done, now you should import from the "Username" / "Sessions" tab.  There is a new "role" in the data, which should be "GM" or "Player"
